addons/netfox/state-synchronizer.gd

```diff
@@ -30,7 +30,7 @@ func _ready():
 func _after_tick(_dt, tick):
 	if is_multiplayer_authority():
 		# Submit snapshot
-		var state = PropertySnapshot.extract(_props)
+		var state: Dictionary = PropertySnapshot.extract(_props)
 		_submit_state.rpc(state, tick)
```

Comment:
I did not update this class at all, because it seems old and probably deprecated. If you wish me to pass the serialized state here as well, I will do it, its easy after all.